### ISSUE TITLE
Add a tip-type callout to the notes shortcode

### ISSUE THREAD

**Issue by cnunciato** [pulumi/pulumi-hugo#970]
We have info-style notes and warnings, but it'd be nice to have tips, too -- e.g., for things like this, which are intended to be more as "try this" than "bear in mind":

![image](https://user-images.githubusercontent.com/274700/149406465-05f1ec56-2f11-47aa-8e94-e714dd840621.png)

We should probably also rename the shortcode from `notes` to something more generic. Leanpub uses an expression called `{blurb}` for this, for example, that takes `class`es like `tip`, `info`, etc., and it works pretty well.

**Comment by cnunciato** [comment]
@susanev If y'all have recommendations for color or icon here, I'd appreciate it! (Was thinking just the usual light-bulb sort of thing.)

**Comment by susanev** [comment]
ill take a look tomorrow morning

**Comment by susanev** [comment]
im a bit confused on what the difference between a note and a tip is, can you give me a bit more information. after we have definitions, it might also be good to go through and review the current notes and change to tips if needed.

for the tip icon, yup, a lightbulb sounds good to me!

separate question, what would you say is the priority order of how important each type is? is a note more important than a tip? having this clarity can help me pick some colors here.

**Comment by cnunciato** [comment]
Sure. The example I shared wasn't the greatest, yeah, it just happened to be at hand when the thought occurred to me. 

The difference is subtle but you can hear it when it's there. A note is like saying "keep in mind" or "take note" -- it's semi-serious, for pointing out things the reader probably shouldn't miss (but are different from warnings, which are like notes that if ignored, could result in something bad happening). A tip is just a suggestion -- I like the dictionary's definition of "a small but useful piece of practical advice." Examples:

```
Note: The Go SDK requires Go 1.16 or greater.
```

This is like saying, "Take note: You need to have Go 1.16 for this to work."  

```
Tip: To bypass the prompt, you can also pass the `-y` option.
```

This reads more like, "Here's a thing you might appreciate. Use it if you like, hope it helps," etc.

So in order of importance, it'd be tip (useful and totally optional), note (important but not dangerous) and warning (bad things could happen). Does that help?

**Comment by susanev** [comment]
yes, thank you!! will prolly not be able to get you color recs until next week, that okay?

**Comment by cnunciato** [comment]
Totally, no rush at all.

**Comment by susanev** [comment]
apologies for the delay on this!

my recommendation is to use a border instead of fill, to avoid introducing a new color here.

[ux designs in figma](https://www.figma.com/file/meIG1AIk1I7Vd56bRIdDMD/note-styles?node-id=0%3A1)

screenshot:
<img width="1199" alt="note-styles" src="https://user-images.githubusercontent.com/5489125/151631165-afa07e8d-ff8b-410a-bb39-3f527888573b.png">
